Source/UETorch/Private/UETorch.cpp

```diff
@@ -973,4 +973,65 @@ extern "C" bool DestroyActor(AActor* object) {
 	}
 	object->K2_DestroyActor();
 	return true;
+}
+
+/**
+ * Functions to control pawns
+ */
+
+extern "C" bool SimpleMoveToLocation(UObject* _this, AActor* object, float x, float y, float z) {
```

Comment:
I guess it would depend on whether the people who would use UETorch intend to use it for animated actors, if it's only used to simulate physics with non-animated actors then I guess it wouldn't make much sense to include it.